Installation fails on ubuntu 18 (any bird)

## AttributeError: 'module' object has no attribute 'SetuptoolsVersion'

## Way to reproduce:
```
"/home/sthao/anaconda/bin/conda" install -y -n malleefowl setuptools=36.5.0
Solving environment: done


==> WARNING: A newer version of conda exists. <==
  current version: 4.4.11
  latest version: 4.5.5

Please update conda by running

    $ conda update -n base conda



## Package Plan ##

  environment location: /home/sthao/.conda/envs/malleefowl

  added / updated specs:
    - setuptools=36.5.0


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    certifi-2018.4.16          |           py27_0         142 KB

The following packages will be UPDATED:

    certifi:         2018.4.16-py27_0 conda-forge --> 2018.4.16-py27_0
    openssl:         1.0.2o-0         conda-forge --> 1.0.2o-h20670df_0

The following packages will be DOWNGRADED:

    ca-certificates: 2018.4.16-0      conda-forge --> 2018.03.07-0    


Downloading and Extracting Packages
certifi 2018.4.16: ################################################################################################################################################################################# | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
Update pinned conda packages ...
Update buildout bootstrap-buildout.py ...
Bootstrap buildout ...
ez_setup.py is deprecated and when using it setuptools will be pinned to 33.1.1 since it's the last version that supports setuptools self upgrade/installation, check https://github.com/pypa/setuptools/issues/581 for more info; use pip to install setuptools
Traceback (most recent call last):
  File "bootstrap-buildout.py", line 200, in <module>
    import zc.buildout.buildout
  File "/tmp/bootstrap-bGD7T9/zc.buildout-2.10.0-py2.7.egg/zc/buildout/buildout.py", line 18, in <module>
    import zc.buildout.easy_install
  File "/tmp/bootstrap-bGD7T9/zc.buildout-2.10.0-py2.7.egg/zc/buildout/easy_install.py", line 45, in <module>
    pkg_resources.SetuptoolsVersion(setuptools_version) >=
AttributeError: 'module' object has no attribute 'SetuptoolsVersion'
Makefile:184: recipe for target 'bootstrap' failed
make: *** [bootstrap] Error 1
```
May be related to: https://github.com/buildout/buildout/issues/442
But works fine on ubuntu16 but not with ubuntu18...

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-VERSION := 0.3.14
+VERSION := 0.3.17
 RELEASE := master
 
 # Include custom config if it is available
@@ -14,8 +14,8 @@ CPU_ARCH := $(shell uname -m 2>/dev/null || uname -p 2>/dev/null || echo "unknow
 
 # Python
 SETUPTOOLS_VERSION := 36.5.0
-CONDA_VERSION := 4.4
-BUILDOUT_VERSION := 2.10.0
+CONDA_VERSION := 4.5
+BUILDOUT_VERSION := 2.12.1
 
 # Anaconda
 ANACONDA_HOME ?= $(HOME)/anaconda
@@ -42,10 +42,6 @@ endif
 # Buildout files and folders
 DOWNLOAD_CACHE := $(APP_ROOT)/downloads
 BUILDOUT_FILES := parts eggs develop-eggs bin .installed.cfg .mr.developer.cfg *.egg-info bootstrap-buildout.py *.bak.* $(DOWNLOAD_CACHE)
-
-# Docker
-DOCKER_IMAGE := birdhouse/$(APP_NAME)
-DOCKER_CONTAINER := $(APP_NAME)
 
 # end of configuration
 
@@ -84,9 +80,6 @@ help:
 	@echo "  stop        to stop supervisor service."
 	@echo "  restart     to restart supervisor service."
 	@echo "  status      to show supervisor status"
-	@echo "\nDocker targets:"
-	@echo "  Dockerfile  to generate a Dockerfile for $(APP_NAME)."
-	@echo "  dockerbuild to build a docker image for $(APP_NAME)."
 
 .PHONY: version
 version:
@@ -103,7 +96,6 @@ info:
 	@echo "  APP_NAME            $(APP_NAME)"
 	@echo "  APP_ROOT            $(APP_ROOT)"
 	@echo "  DOWNLOAD_CACHE      $(DOWNLOAD_CACHE)"
-	@echo "  DOCKER_IMAGE        $(DOCKER_IMAGE)"
 
 ## Helper targets ... ensure that Makefile etc are in place
 
@@ -294,26 +286,3 @@ restart:
 status:
 	@echo "Supervisor status ..."
 	bin/supervisorctl status
-
-
-## Docker targets
-
-.PHONY: Dockerfile
-Dockerfile: bootstrap
-	@echo "Update Dockerfile ..."
-	bin/buildout -c custom.cfg install docker
-
-.PHONY: dockerrmi
-dockerrmi:
-	@echo "Removing previous docker image ..."
-	docker rmi $(DOCKER_IMAGE)
-
-.PHONY: dockerbuild
-dockerbuild: Dockerfile
-	@echo "Building docker image ..."
-	docker build --rm -t $(DOCKER_IMAGE) .
-
-.PHONY: dockerrun
-dockerrun: dockerbuild
-	@echo "Run docker image ..."
-	docker run -i -t -p 9001:9001 --name=$(DOCKER_CONTAINER) $(DOCKER_IMAGE) /bin/bash

--- a/profiles/versions.cfg
+++ b/profiles/versions.cfg
@@ -6,8 +6,8 @@ birdhousebuilder.recipe.sphinx = 0.2.3
 buildout.locallib = 0.3.1
 pluggy = 0.6.0
 py = 1.5.2
-zc.buildout = 2.10.0
-zc.recipe.egg = 2.0.5
+zc.buildout = 2.12.1
+zc.recipe.egg = 2.0.6
 
 # Required by:
 # birdhousebuilder.recipe.nginx==0.3.6

